server/sonar-server/src/main/java/org/sonar/server/organization/ws/DeleteAction.java

```diff
@@ -31,21 +31,16 @@
 import org.sonar.server.user.UserSession;
 
 import static com.google.common.base.Preconditions.checkArgument;
-import static java.lang.String.format;
-import static org.sonar.core.util.Uuids.UUID_EXAMPLE_03;
-import static org.sonar.server.organization.ws.OrganizationsWsSupport.PARAM_ID;
 import static org.sonar.server.organization.ws.OrganizationsWsSupport.PARAM_KEY;
 
 public class DeleteAction implements OrganizationsAction {
   private static final String ACTION = "delete";
 
-  private final OrganizationsWsSupport wsSupport;
   private final UserSession userSession;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "wsSupport" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1068)